addons/send/readme.txt

```diff
@@ -5,3 +5,17 @@ If you wish to change the color of the text added from aecho. Or any plugin that
 windower.add_to_chat(55,msg:sub(5))
 
 You may change the 55 to any number from 1 to 255 to get a (not always) different color. 
+
+Can use <tid> (stands for target id) from the user originating the send and it will replace the <tid> with the senders currently targeted mob id.
+
+/con send @others /ma "Blizzard IV" <tid>
+
+and another program, e.g. gearswap will resolve the target id to the monster and cast blizzard IV on the mob targeted by the sender. 
```

Comment:
I would mention that GS is strictly required for any ID replacement to work. I'd call it an `addon` instead of a `program` though, just to avoid confusion among non-technical users.